### DISCUSSION TITLE
Reverse proxy requirements

### DISCUSSION THREAD

**Discussion by geiseri** [coder/coder#4264] [General]
Hi, I have a traefik server that I use on my bastion server to do SSL and dispatch to internal services.  If I point it at my coder install and then set the `CODER_ACCESS_URL` to the dns name of the bastion server's public domain I get a UI.  The issue is once I try to use a terminal, ssh, or code-server It never seems to connect.  Is there anything special that needs to happen if the server that is hosting coder is different than the server hosting the reverse proxy?

**Comment by matifali** [comment]
All the workspaces should be able to connect to the `CODER_ACCESS_URL`
